source/deploy/mobile-app-changelog.md

```diff
@@ -37,6 +37,7 @@ This changelog summarizes updates to Mattermost mobile apps releases for [Matter
  - Added ``@formatjs/intl-listformat`` and ``path-to-regexp``, and removed ``@nozbe/with-observables``, ``@react-native/eslint-config`` and ``@react-native/metro-config`` from https://github.com/mattermost/mattermost-mobile.
 
 ### Known Issues
+ - Popovers, such as emoji selectors and the channel context menu, may be non-functional in the mobile app [MM-56582](https://mattermost.atlassian.net/browse/MM-56582). A workaround is to disable **Reduce Motion** in your phone settings and then re-launching the app.
```

Comment:
When I read this, I initially interpreted "motion" to mean physical motion of the device itself, not "motion" displayed on the screen. If we intend the latter, perhaps we want to mention that the **Reduce Motion** device option is related to accessibility.